### PR TITLE
Add 08-03-2020 TSC meeting notes

### PR DESCRIPTION
**Pull request by michdolan**

### CODE REVIEW

File: docs/guides/contributing/tsc/meetings/2020-08-03.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      community. Static site for gh-pages. Talking design ideas with Mario, but 
+      will get something up for now.
+    - Michael: Will have PR in by end of week with documentation sections 
+      committed to.
+    - Sean: Anyone able to get docs built?
+    - Doug: Requires Python 3 so a bit problematic. Requirement for Sphinx 2, a 
+      theme dependency, which is Python 3 only. 
+    - Any other open infastructure tasks?
+    - Sean: Still working on API doc build. Current PR is for extracting 
+      docstrings from python binding, blocked by a clang depednency.
```
Comment (Simran-B) [inline docs/guides/contributing/tsc/meetings/2020-08-03.md:47]:
```suggestion
      docstrings from python binding, blocked by a clang dependency.
```

File: docs/guides/contributing/tsc/meetings/2020-08-03.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    - Carol: Could talk angle of community involvement with docs.
+    - Michael: Can also have brief intro to docs to get interest.
+    - Doug: Good to have more participation in the BoF.
+    - Carol: Sean and I can talk and have slides about process and introducing 
+      docs. If there's no Q&A, good to introduce landing point for exploring 
+      docs. More important if there's no discussion.
+    
+* OCIO v2 WG meeting
+    - Doug: Will this Thursday work? Will send email. Number of PRs going to 
+      land soon. Want to give explanation and guidance about the work. These PRs
```
Comment (doug-walker) [inline docs/guides/contributing/tsc/meetings/2020-08-03.md:73]: Please change "These PRs are remaining work to be feature complete" to "Will discuss the work remaining to be feature complete."

File: docs/guides/contributing/tsc/meetings/2020-08-03.md (hunk 1 above)
Comment (michdolan) [reply docs/guides/contributing/tsc/meetings/2020-08-03.md]: Fixed.

File: docs/guides/contributing/tsc/meetings/2020-08-03.md (hunk 2 above)
Comment (michdolan) [reply docs/guides/contributing/tsc/meetings/2020-08-03.md]: Changed.

**Pull request merged**